charts/prometheus-redis-exporter/README.md

```diff
@@ -50,6 +50,42 @@ $ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 5.0.0
+
+From 5.0.0 redis exporter is using this above as selector labels in service and deployments
+```
+app.kubernetes.io/name: {{ include "prometheus-redis-exporter.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+```
+
+selector labels is immutable in kubernetes, so uninstall is needed if you're upgrading from 4.8.0 and below
+
+
+```console
+helm uninstall {{ .Release.Name }}
```

Comment:
You could keep your helm history if you delete the deployment before upgrading:

```suggestion
kubectl delete deployment -l app=prometheus-redis-exporter
helm upgrade -i prometheus-redis-exporter prometheus-community/prometheus-redis-exporter
```

Reply:
Thank you! I didn't know that could be done